File: BOJ/Bronze/B_cpp/2576 홀수.cpp
#include <iostream>
using namespace std;

int main()
{
	int n;
	int minOdd = 100;
	int sumOdd = 0;

	for (int i = 0; i < 7; i++) {
		cin >> n;

		if (n % 2 == 1) {
			sumOdd += n;
			if (n < minOdd)
				minOdd = n;
		}
	}

	if (sumOdd == 0)
		cout << -1;
	else {
		cout << sumOdd << "\n" << minOdd;
	}
	
	return 0;
}

File: BOJ/python/2576 홀수.py
min_odd = 100
sum_odd = 0

for _ in range(7):
    n = int(input())
    if n % 2 == 1:
        sum_odd += n
        if n < min_odd:
            min_odd = n

if sum_odd == 0:
    print(-1)
else:
    print("{}\n{}".format(sum_odd, min_odd))
